stdlib/2/__builtin__.pyi

```diff
@@ -482,6 +483,8 @@ class tuple(Sequence[_T_co], Generic[_T_co]):
     def __le__(self, x: Tuple[_T_co, ...]) -> bool: ...
     def __gt__(self, x: Tuple[_T_co, ...]) -> bool: ...
     def __ge__(self, x: Tuple[_T_co, ...]) -> bool: ...
+    def __eq__(self, x: Tuple[_T_co, ...]) -> bool: ...
+    def __ne__(self, x: Tuple[_T_co, ...]) -> bool: ...
```

Comment:
Since this issue affects other type checkers as well, not just mypy, I created a typing issue where we can discuss this: https://github.com/python/typing/issues/378